rosetta/validator/validator.go

```diff
@@ -27,12 +27,10 @@ type Validator struct {
 }
 
 func New(params dps.Params, index dps.Reader, config Configuration) *Validator {
-
 	v := &Validator{
 		params:   params,
 		index:    index,
 		validate: newRequestValidator(config),
 	}
-
 	return v
 }
```

Comment:
IMO the first newline can go, but the second one should remain. The return should not cuddle with the definition because it is a multi-line one. See https://github.com/bombsimon/wsl/blob/master/doc/rules.md#return-statements-should-not-be-cuddled-if-block-has-more-than-two-lines

If you can, please remove those changes entirely from this PR as they will conflict with https://github.com/optakt/flow-dps/pull/449 which changes this function as well.